blog/2024-12-18-transportation-to-ga.mdx

```diff
@@ -0,0 +1,50 @@
+
+First, let’s talk about what GA means for the transportation theme. We have a few guiding principles:
+
+1. There are no "trust violations" in the schema and data. It needs to "just work" when a developer writes code based on our schema and documentation.
+2. The road geometry and attributes should be on par with the roads in OSM.
+3. Roads have stable GERS IDs and those IDs can be references for attaching additional data.
+4. The road features we capture should support our core use cases: rendering for display, GERS referencing, and basic routing capabilities.
+
+Most of our work over the past few months has been around #1 and #2. We've been resolving data-schema violations by fixing inconsistencies in our schema and issues with the data transformation steps in our pipeline. To satisfy #3, we’ve been monitoring changes in GERS IDs from release to release, and our metrics show us that the vast majority of roads are maintaining stable IDs.
+
+We've had to break a few things in our schema to make way for improvements:
+
+- To better support routing use cases, we added to the segment feature type a new property called `connectors`: an array of IDs with pre-computed linear reference values that explicitly link segments and connector features via coordinates. Each connector is a possible routing decision point, meaning it defines a place along the segment in which there is possibility to transition to other segments that share the same connector. This new property — a replacement for the old `connector_ids` property — will reduce complexity and improve accuracy as we calculate the intersections between segment and connector features. You can read more about this improvement [here](https://docs.overturemaps.org/blog/2024/08/15/preview-august-release/#transportation-schema-changes).
+
+- In the [November release](https://docs.overturemaps.org/release/2024-11-13.0/), we removed `lanes` from the schema to eliminate a significant trust violation. The `lanes` property has existed in the schema for months but we never populated it with data in our releases, which is a poor experience for our community. We intend to redesign `lanes` in the coming months.
+
+With these changes, we are confident we’ve met our requirements for general availability (GA). We'll continue refining our schema and improving our data throughout 2025. We encourage our community to keep the feedback coming, particularly around data quality and our core use cases of map display, routing, and matching data via GERS.
+
+Here are some specific ways you can help:
+
+1. Examine transportation data in your areas of interest and compare it to local sources of data. Here are some suggestions for downloading data using Overture's Python CLI.
```

Comment:
Worth a link to the CLI tool? https://docs.overturemaps.org/getting-data/overturemaps-py/